kordac/tests/FrameTest.py

```diff
@@ -7,8 +7,15 @@
 from kordac.tests.ProcessorTest import ProcessorTest
 
 class FrameTest(ProcessorTest):
+    '''The iframe processor inherits from the generic tag procesor.
+    The tests contained here test that arguments and the output
+    (html-template) work as expected.
+    '''
 
```

Comment:
In a black-box situation I'd agree with you. In the white-box situation due to this being a generic tag all expected functionality is essentially tested here.